cupy/cudnn.py

```diff
@@ -326,3 +328,11 @@ def add_tensor(handle, alpha, biasDesc, biasData, beta, srcDestDesc,
                srcDestData):
     cudnn.addTensor_v3(handle, alpha, biasDesc,
                        biasData, beta, srcDestDesc, srcDestData)
+
+
+def is_tensor_core_available(dtype):
+    if (dtype is numpy.dtype(numpy.float16) and
```

Comment:
Please remove `if.
Please use `numpy.float16 == dtype`,
Please check the conditions with small calculation amount first.

Reply:
OK, will fix to use `numpy.float16 == dtype`.